plugin/Model/FormModelBuilder/FormModelBuilderDBEstateUnitListSettings.php

```diff
@@ -234,4 +235,20 @@ public function getInputModelCustomLabelLanguageSwitch(): InputModelDB
 		});
 		return $pInputModel;
 	}
+
```

Comment:
This function already exist in FormModelBuilderDBEstateListSettings
Why would you duplicate it here?

Reply:
Sorry for the confusion. The function was indeed duplicated, and I have updated the code to remove it. Could you please review it again? Thank you.

Reply:
No problem @dai-eastgate :)
Could you first take a look at the unit tests please?

Reply:
@andernath The unit tests have been addressed in the PR at [oo-wp-plugin#792](https://github.com/onOffice-Web-Org/oo-wp-plugin/pull/792). Could you please review it first?

Reply:
@dai-eastgate Ops sorry I my fault, missed your comment from yesterday. Of course I will start reviewing fist. :)